docs/intro.md

```diff
@@ -22,23 +22,26 @@ docker pull ghcr.io/cucapra/calyx:latest
 docker run -it --rm ghcr.io/cucapra/calyx:latest
 ```
 
-If you're using the container, skip to [Running Core Tests](#running-core-tests).
+The `--rm` flag will remove the container after you exit the shell. If you want to keep the container around, remove the flag.
 
-**Note**: The `--rm` flag will remove the container after you exit the shell. If you want to keep the container around, remove the flag.
+> If you choose to use the Docker container, you can skip a lot of what follows and head to [*running a hardware design*][hw-design].
```

Comment:
We should make people run core tests as a sanity check

Reply:
Done! The `mrxl` tests do take a minute, although there are only 6 of them. Pretty sure people will start raising their hands and wondering if everything is okay, haha. So if it's okay, I'd rather keep it to the core stuff!

Reply:
I see, FWIW if you runt `runt -v`, it prints out the passing tests eagerly so it looks like its making progress (although the counters update live). One argument for having the `mrxl` tests running is that if there are any problems at that point, we can catch them early instead of when people get to the MrXL tutorial